### ISSUE TITLE
VPN Traffic Split

### ISSUE THREAD

**Issue by enavajas** [mozilla/bedrock#13558]
Hi! This request re VPN came from Dan. At this point, we are good moving forward with sending 100% of traffic for en_US to the new variant, but they would like to test a horizontal variant as well.

The scope of the following ask is:
- Move variant 1 (old) to 0%
- Keep variant 2 (new) at 50%
- Add variant 3 (with horizontal pricing) to 50%

This would allow us to still have a level of mitigation while we move towards confidence in the new designs. 

We will need to end the current test and configure a new one to test horizontal vs. vertical on the LP. 

In a nutshell: 100% of en-US gets the refreshed design, and that audience is split 50/50 between the two pricing layouts. 

Please let me know if you have any questions. Thank you :)

**Comment by craigcook** [comment]
We did the first portion of this task: which is removing the previous experiment and showing the refreshed page to 100% of English. The second part is setting up a new experiment to show the two different pricing layouts and measure which converts better.
